empress_wrapper_examples.py

```diff
@@ -0,0 +1,74 @@
+# TODO: To be converted to automated tests
+# https://github.com/ssantichaivekin/eMPRess/issues/32
+import empress
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg
+import matplotlib.pyplot as plt
+import tkinter
+
+# Read Reconciliation Input
+recon_input = empress.read_tree("./example/heliconius.newick")
+
+# Display Costscape Window where we can click on x/y axis
+class TkinterCostPolygonPage:
+    """Put this class in your app"""
+    def __init__(self):
```

Comment:
I haven't tested whether something like this will work with retrieving the the x/y position on click.